chaincfg/regnetparams.go

```diff
@@ -349,6 +349,33 @@ func RegNetParams() *Params {
 				},
 				StartTime:  0,             // Always available for vote
 				ExpireTime: math.MaxInt64, // Never expires
+			}, {
+				Vote: Vote{
+					Id:          VoteIDAutoRevocations,
+					Description: "Enable automatic ticket revocations as defined in DCP0009",
+					Mask:        0x0006, // Bits 1 and 2
```

Comment:
Bits 5 and 6 (also the individual bits in the choices)

Reply:
Updated here and the individual bits.